src/datasources/asset/types.ts

```diff
@@ -0,0 +1,140 @@
+import { DataQuery } from '@grafana/schema'
```

Comment:
General question about this api, is everything defined here being used? For e.g., the only filters I've seen in the code is around location and workspace. If that is the case, I think this file can be simplified

Reply:
everything will be used for features that already done but not exists in current PR